Makefile

```diff
@@ -2,7 +2,7 @@ CC ?= gcc
 CFLAGS = -std=c99 -Os -Wall -Wextra
 
 BIN = jvm
-OBJS = jvm.o stack.o constant-pool.o classfile.o
+OBJS = jvm.o stack.o constant-pool.o classfile.o class_heap.o
```

Comment:
Since you are going to build more files, multi-line would be better for change tracking. That is, use the following
```
OBJS = \
	jvm.o \
	stack.o \
	constant-pool.o \
	classfile.o \
	class_heap.o
```